test/README.md

```diff
@@ -0,0 +1,210 @@
+# Testing in Crawlee with vitest
```

Comment:
i'd say this should go to the contributing guide instead

Reply:
I mostly went with adding it here for visibility, but I can move it to the contributing guide (or add a reference to this from there instead?)

Reply:
contributing guide is where this information should be, having it elsewhere imo reduces visibility, as you have something here and something elsewhere. let's move it there to have everything in place place.